docs/jep/18137-numpy-scipy-scope.md

```diff
@@ -0,0 +1,345 @@
+# JEP 18137: Scope of JAX NumPy & SciPy Wrappers
+
+*Jake VanderPlas*
+
+*October 2023*
+
+Until now, the intended scope of {mod}`jax.numpy` and {mod}`jax.scipy` has been relatively
+ill-defined. This document proposes a well-defined scope for these packages to better guide
+and evaluate future contributions, and to motivate the removal of some out-of-scope code.
+
+## Background
+From the beginning, JAX has aimed to provide a NumPy-like API for executing code in XLA,
+and a big part of the project’s development has been building out the {mod}`jax.numpy` and
+{mod}`jax.scipy` namespaces as XLA-backed wrappers of NumPy and SciPy APIs. There has always
```

Comment:
```suggestion
{mod}`jax.scipy` namespaces as JAX-based implementations of NumPy and SciPy APIs. There has always
```

... since they support all jax transformations.